### ISSUE TITLE
qgis metasearch traffic not logged

### ISSUE THREAD

**Issue by pvgenuchten** [rduivenvoorde/qgisnetworklogger#32]
this is probably a problem of metasearch, seems this plugin does not use generic libs to connect to external resources, however want to bring this up as an example, this may happen for other plugins as well

**Comment by rduivenvoorde** [comment]
Yes, metasearch does not use Qt's (and with that QGIS') build-in machinery to do requests.

So QGIS is not aware of those, and a plugin cannot log it. Same goes for certain GDAL-requests (which go via GDAL directly instead of via the QgsNetworkManager classes.

This plugin depends on signals received from QgsNetworkManager classes that a request is fired, returned or failed.
